app/templates/components/public/speaker-list.hbs

```diff
@@ -0,0 +1,6 @@
+<h3>{{t 'Speakers'}}</h3>
+<div class="speaker-list ui grid">
+  {{#each speakers as |speaker|}}
+    {{ public/speaker-item speaker=speaker}}
```

Comment:
No space around the component. `{{public/speaker-item speaker=speaker}}`. Will add all this information in the linter and a style doc soon.